frontend/appflowy_flutter/lib/mobile/presentation/setting/font/font_setting.dart

```diff
@@ -38,7 +40,9 @@ class FontSetting extends StatelessWidget {
         if (newFont != null && newFont != selectedFont) {
           if (context.mounted) {
             context.read<AppearanceSettingsCubit>().setFontFamily(newFont);
-            context.read<DocumentAppearanceCubit>().syncFontFamily(newFont);
+            unawaited(
+              context.read<DocumentAppearanceCubit>().syncFontFamily(newFont),
+            );
```

Comment:
I think the cleanup is good, but the lint forces us to use this `unawaited`, which I don't particularly like 😂

Reply:
Yes, that's the downside.

Reply:
HAHA. I agreed with @Xazin . However, there may be a few rare instances where `unawaited` is necessary